src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -177,11 +178,16 @@ export default {
                 return;
             }
 
+            let value;
             if (this.item.format) {
-                return printj.sprintf(this.item.format, this.datum[this.valueMetadata.key]);
+                value = this.customformatter.format(this.datum);
```

Comment:
Again, `valueFormatter` and `customFormatter` *have the same interface*. We don't need two variables to track it, we can just have `this.formatter`.

Reply:
Can we have a call, not sure what exactly to change here?

Reply:
updated.